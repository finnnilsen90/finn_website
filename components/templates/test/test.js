const React = require('react');
const ReactTestUtils = require('react-dom/test-utils');
const renderer = require('react-test-renderer')

const VarName = require('../compName.js');
const ChildConst = require('../lib/child/child.js')

let varNameConst = new VarName(); 

jest.mock('../lib/child/child.js', () => 'ChildConst')

describe('component connection', () => {

    it('should be defined', () => {
        expect(VarName).toBeDefined();
    })

})  
   
describe('component content', () => {

it('should match snapshot of test div.', () => {
     const component = renderer.create(
        <div className='containter'>
            <Libcomp />
        </div>          
        )
        let tree = component.toJSON();
        expect(tree).toMatchSnapshot();
    });

})
