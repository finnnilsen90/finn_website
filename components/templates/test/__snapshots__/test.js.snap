// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`component content should match snapshot of test div. 1`] = `
<div
  className="containter"
>
  <Libcomp />
</div>
`;
